keg/image_definition.py

```diff
@@ -15,25 +15,39 @@
 # You should have received a copy of the GNU General Public License
 # along with keg. If not, see <http://www.gnu.org/licenses/>
 #
-from datetime import datetime, timezone
 import os
+from typing import (
+    List, Dict
+)
+from datetime import (
+    datetime, timezone
+)
 
+# project
 from keg import version, utils
 from keg.exceptions import KegError
 
 
-class ImageDefinition:
-    """Class for constructing a keg image definition from recipes"""
-
-    def __init__(self, image_name, recipes_root, data_roots):
-        """Init ImageDefintion with image_name and recipes root path"""
+class KegImageDefinition:
+    """
+    Class for constructing a keg image definition from recipes
+    """
+    def __init__(
+        self, image_name: str, recipes_root: str, data_roots: List = []
```

Comment:
do we know the type of the content for `data_roots` list? Probably `List[str]`

Reply:
yes I think this can be done

Reply:
done